backend/src/test/java/shook/shook/song/domain/repository/SongRepositoryTest.java

```diff
@@ -78,4 +81,29 @@ void createdAt_prePersist() {
         assertThat(song).isSameAs(saved);
         assertThat(song.getCreatedAt()).isBetween(prev, after);
     }
+
+    @DisplayName("모든 노래와 노래의 총 득표수를 가진 객체 리스트를 반환한다.")
+    @Test
+    void findSongAndTotalVoteCount() {
+        //given
+        //when
+        final List<SongTotalVoteCountDto> songWithTotalVoteCount = songRepository.findSongWithTotalVoteCount();
+
+        //then
+        final List<Song> songs = songWithTotalVoteCount.stream()
+            .map(SongTotalVoteCountDto::getSong)
+            .toList();
+        final List<Long> totalVotes = songWithTotalVoteCount.stream()
+            .map(SongTotalVoteCountDto::getTotalVoteCount)
+            .toList();
+
+        final List<Song> expectedSongs = songRepository.findAll();
+        final List<Long> expectedTotalVotes = IntStream.range(0, 40)
+            .mapToObj((index) -> 0L)
+            .toList();
+
+        assertThat(songs).usingRecursiveComparison().isEqualTo(expectedSongs);
+        assertThat(totalVotes).usingRecursiveComparison().isEqualTo(expectedTotalVotes);
```

Comment:
다른 테스트 코드처럼 한 줄에 하나의 . 이 찍히게 통일하는건 어떨까용 ㅎ.ㅎ

```suggestion
        assertThat(songs).usingRecursiveComparison()
          .isEqualTo(expectedSongs);
        assertThat(totalVotes).usingRecursiveComparison()
          .isEqualTo(expectedTotalVotes);
```